packages/commcare/src/Adaptor.js

```diff
@@ -316,6 +316,37 @@ export function fetchReportData(reportId, params, postUrl) {
   };
 }
 
+/**
+ * Make a general HTTP request against the Commcare server.
+ * @example <caption>Make a GET request to get cases</caption>
+ * request("GET", "/a/asri/api/v0.5/case");
+ * @function
+ * @public
+ * @param {string} method - HTTP method to use
+ * @param {string} path - Path to resource
+ * @param {object} body - Object which will be attached to the body
+ * @param {RequestOptions} options - Optional request params
+ * @returns {Operation}
+ * @state {CommcareHttpState}
+ */
+export function request(method, path, body, options = {}) {
+  return async state => {
+    const [resolvedMethod, resolvedPath, resolvedBody, resolvedOptions] =
+      expandReferences(state, method, path, body, options);
+    try {
+      const response = await util.request(state.configuration, resolvedPath, {
+        method: resolvedMethod,
+        data: resolvedBody,
+        params: resolvedOptions,
+        contentType: 'application/json',
+      });
+
+      return util.prepareNextState(state, response);
+    } catch (e) {
```

Comment:
You can remove this try/catch block as we're not doing any processing on the error